django/core/management/templates.py

```diff
@@ -81,9 +87,13 @@ def handle(self, app_or_project, name, target=None, **options):
                                    "exist, please create it first." % top_dir)
 
         extensions = tuple(handle_extensions(options['extensions']))
-        extra_files = []
-        for file in options['files']:
-            extra_files.extend(map(lambda x: x.strip(), file.split(',')))
+        extra_files = [
+            x.strip() for file in options['files'] for x in file.split(',')
+        ]
+        exclusions = [
+            x.strip() for dir in options['exclusions'] for x in dir.split(',')
```

Comment:
Since `-x` can already be used multiple times, is it necessary to support the comma-delimited variant? It introduces the edge case of paths containing commas that I think should be addressed somehow, e.g. either by documenting the limitation or handling and testing that case.

Reply:
This applies to `--name` as well and I think it's best to make them similar. Changing `--name` would be backwards-incompatible, so I think we should just document it.

Reply:
Reflecting on this a bit more, not allowing directories with commas in their names to be excluded seems like an artificial and unnecessary restriction. `--exclude` is a new option and doesn't have to match `--name` (which was added ~10 years ago).

Also, looking through the code base, it looks like there are a number of commands with an `--exclude` or `--exclude`-like option that support being used multiple times, but don't also support separating multiple values with commas (even for cases where we know the values can't have commas):

`compilemessages`: https://github.com/django/django/blob/54a30a7a00fea6c5e3702282ade6e0238e06de3b/django/core/management/commands/compilemessages.py#L44-L47

`dumpdata`: https://github.com/django/django/blob/54a30a7a00fea6c5e3702282ade6e0238e06de3b/django/core/management/commands/dumpdata.py#L53-L57

`loaddata`: https://github.com/django/django/blob/54a30a7a00fea6c5e3702282ade6e0238e06de3b/django/core/management/commands/loaddata.py#L60-L63

`makemessages`: https://github.com/django/django/blob/54a30a7a00fea6c5e3702282ade6e0238e06de3b/django/core/management/commands/makemessages.py#L222-L225

`test` command: https://github.com/django/django/blob/54a30a7a00fea6c5e3702282ade6e0238e06de3b/django/test/runner.py#L635-L638

Reply:
Fair enough, I've removed the limitation.